### PR TITLE
qw configure

### PR DESCRIPTION
**Pull request by tim-band**

### CODE REVIEW

**Review by tim-shifty — APPROVED** [review]

**Review by tim-shifty — APPROVED** [review]
Looks good!
-Socky

File: .github/workflows/qw-pr-check.yml
Comment (tim-shifty) [inline .github/workflows/qw-pr-check.yml:1]: comment on things here.

**Review by tim-band — COMMENTED** [review]
It's OK

File: .github/ISSUE_TEMPLATE/requirement.yml
Comment (tim-shifty) [inline .github/ISSUE_TEMPLATE/requirement.yml:1]: I like this one.

**Review by tim-shifty — APPROVED** [review]
Do I have to click "viewed" for each file?

File: .github/ISSUE_TEMPLATE/requirement.yml
Comment (tim-band) [reply .github/ISSUE_TEMPLATE/requirement.yml:1]: Yes, nice. I shall resolve.

**Review by tim-shifty — APPROVED** [review]
In response to the request, I approve!

**Pull request merged**